README.md

```diff
@@ -172,6 +181,35 @@ AddLiquidity.v1_1.execute();
 
 <hr>
 
+### Working with the local copy of the sdk
```

Comment:
We can create a CONTRIBUTING.md and add these details there. I think it would be better to have a dedicated file that explains how one might contribute to the library